api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java

```diff
@@ -48,39 +48,30 @@ public UpcomingBirthdayEmailHandler(@Value("${payment-cycle.number-of-calculatio
+        EmailMessagePayload messagePayload = buildChildTurnsAgeNotificationEmailPayload(paymentCycle,
+                entitlement, multipleChildrenTurningFourInNextMonth, EmailType.CHILD_TURNS_FOUR);
         log.info("Sending email for child turns 4 for Payment Cycle after cycle with id: [{}]", paymentCycle.getId());
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
     }
 
     public void sendChildTurnsOneEmail(PaymentCycle paymentCycle, NextPaymentCycleSummary dateOfBirthSummaryAffectingNextPayment) {
         PaymentCycleVoucherEntitlement entitlement = determineEntitlementForNextCycle(paymentCycle);
         boolean multipleChildrenTurningOneInNextMonth = dateOfBirthSummaryAffectingNextPayment.hasMultipleChildrenTurningOne();
-        EmailMessagePayload messagePayload = buildChildTurnsOneNotificationEmailPayload(paymentCycle, entitlement, multipleChildrenTurningOneInNextMonth);
+        EmailMessagePayload messagePayload = buildChildTurnsAgeNotificationEmailPayload(paymentCycle,
+                entitlement, multipleChildrenTurningOneInNextMonth, EmailType.CHILD_TURNS_ONE);
         log.info("Sending email for child turns 1 for Payment Cycle after cycle with id: [{}]", paymentCycle.getId());
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
     }
 
-    private EmailMessagePayload buildChildTurnsFourNotificationEmailPayload(PaymentCycle paymentCycle,
-                                                                            PaymentCycleVoucherEntitlement entitlementNextMonth,
-                                                                            boolean multipleChildrenTurningFourInMonth) {
-        Map<String, Object> emailPersonalisation = createEmailPersonalisationMapForNextCycle(paymentCycle, entitlementNextMonth);
-        emailPersonalisation.put(MULTIPLE_CHILDREN.getTemplateKeyName(), multipleChildrenTurningFourInMonth);
-        return EmailMessagePayload.builder()
-                .claimId(paymentCycle.getClaim().getId())
-                .emailType(EmailType.CHILD_TURNS_FOUR)
-                .emailPersonalisation(emailPersonalisation)
-                .build();
-    }
-
-    private EmailMessagePayload buildChildTurnsOneNotificationEmailPayload(PaymentCycle paymentCycle,
+    private EmailMessagePayload buildChildTurnsAgeNotificationEmailPayload(PaymentCycle paymentCycle,
                                                                            PaymentCycleVoucherEntitlement entitlementNextMonth,
-                                                                           boolean multipleChildrenTurningOneInMonth) {
+                                                                           boolean multipleChildrenTurningOneInMonth,
+                                                                           EmailType emailType) {
```

Comment:
Parameter should be multipleChildrenTurningAgeInMonth

Reply:
Thanks, have corrected.